ui/src/Components/CreateProfile.jsx

```diff
@@ -25,6 +19,10 @@ class CreateProfile extends React.Component {
             ref1Email: '',
             ref2Name: '',
             ref2Email: '',
+            govIdFlag: false,
```

Comment:
Can govId != '' serve the same purpose to these flags?

Reply:
The reason why I did it like this is bc null checking in JavaScript can be a bit weird at times. You're right that we don't need it, but just thought that it might make our lives a bit easier